extensions/vscode/src/views/homeView.ts

```diff
@@ -1379,10 +1382,15 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
           },
         });
       } catch (error: unknown) {
-        const summary = getSummaryStringFromError(
-          "sendRefreshedFilesLists, files.getByConfiguration",
-          error,
-        );
+        let summary = "";
+        if (isAxiosJsonErrorRes(error)) {
```

Comment:
This is where the suggestion above would change the code, to not require the if statement.